### PR TITLE
Add ECCO dataset

### PR DESCRIPTION
**Pull request by ividito**
Related: https://github.com/NASA-IMPACT/veda-data-pipelines/pull/219 and https://github.com/NASA-IMPACT/veda-data-pipelines/issues/217

Add configuration for ECCO ocean state estimate dataset.

### CODE REVIEW

**Comment by danielfdsilva** [comment]
@ividito Is this ready for review?

**Comment by ividito** [comment]
> @ividito Is this ready for review?

Yes, this should be ready to go.

File: datasets/ecco-surface-height-change.data.mdx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+id: ecco-surface-height-change
+name: "ECCO sea-surface height change from 1992 to 2017"
+description: "Gridded global sea-surface height change from 1992 to 2017 from the Estimating the Circulation and Climate of the Ocean (ECCO) ocean state estimate."
+media:
+  src: ::file ./ecco-surface-height-change--dataset-cover.jpg
```
Comment (danielfdsilva) [inline datasets/ecco-surface-height-change.data.mdx:6]: With 1.8MB this image is pretty big. Consider cutting it down to 1920 on the wide side and running it through http://compressor.io/

File: datasets/ecco-surface-height-change.data.mdx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        - "#00FFFF"
+        - "#00FF00"
+
+---
+
+<Block>
+  <Prose>
+    ## About
+
+    Gridded global sea-surface height change from 1992 to 2017 from the Estimating the Circulation and Climate of the Ocean (ECCO) ocean state estimate. The dataset was calculated as the difference between the annual means over 2017 and 1992, from the 0.5 degree, gridded monthly mean data product available on PO.DAAC (https://podaac.jpl.nasa.gov/dataset/ECCO_L4_SSH_05DEG_MONTHLY_V4R4).
```
Comment (danielfdsilva) [inline datasets/ecco-surface-height-change.data.mdx:42]: Use `PO.DAAC` for the link, instead of showing the actual url

**Review by danielfdsilva — CHANGES_REQUESTED** [review]

**Review by danielfdsilva — APPROVED** [review]
Fantastic! :D

**Comment by j08lue** [comment]
Can we merge this, @ividito?

**Pull request merged**